lib/components/DrawArea/DrawArea.tsx

```diff
@@ -152,33 +168,32 @@ export const DrawArea = React.forwardRef<IHandles, IProps>((props, ref) => {
           />
         </Fade>
       ) : (
-        <Fade in>
-          <svg
-            width="100%"
-            height="100%"
-            viewBox="0 0 100 100"
-            preserveAspectRatio="none"
-            className={css({
-              "& path": {
-                fill: "none",
-                strokeWidth: "5px",
-                stroke: textColors.secondary,
-                strokeLinejoin: "round",
-                strokeLinecap: "round",
-              },
-            })}
-          >
-            {lines.map((line, index) => (
-              <DrawingLine key={index} line={line} />
-            ))}
-          </svg>
-        </Fade>
-      )}
+          <Fade in>
+            <svg
+              width="100%"
+              height="100%"
+              viewBox="0 0 100 100"
+              preserveAspectRatio="none"
+              className={css({
+                "& path": {
+                  fill: "none",
+                  strokeWidth: "5px",
+                  stroke: textColors.secondary,
+                  strokeLinejoin: "round",
+                  strokeLinecap: "round",
+                },
+              })}
+            >
+              {lines.map((line, index) => (
+                <DrawingLine key={index} line={line} />
+              ))}
+            </svg>
+          </Fade>
+        )}
```

Comment:
Yeah using VSCode. I'll check my extensions when I get back to the home setup. I'll just be better about letting that kind of stuff through the commits, since it's a nonimpactful change.